README.md

```diff
@@ -1,208 +1,262 @@
+
+<img src="assets/images/implementation guide/Import Push Notification module as a new module.JPG"/>
+
+### Step 4 - Update component.json file
+
+Find the `theme\components.json` file in your project. This file contains (among others) the dependencies of the to-be-created mobile hybrid application. Update `theme\components.json` by adding `"widgets/pushNotifications/lib/PushNotification.js"` as an element of the `js` array so it would look like this:
+
 ```
-{
-    "files": {
-        "css": ["lib/bootstrap/css/bootstrap.min.css", "mxclientsystem/mxui/ui/mxui.css", "css/theme.css"],
-        "js": ["mxclientsystem/mxui/mxui.js", "js/jquerymin.js", "widgets/pushNotifications/lib/PushNotification.js"]
-    },
-    "cachebust": "{{cachebust}}"
+{  
+   "files":{  
+      "css":[  
+         "lib/bootstrap/css/bootstrap.min.css",
+         "mxclientsystem/mxui/ui/mxui.css",
+         "styles/css/lib/lib.css",
+         "styles/css/custom/custom.css"
+      ],
+      "js":[  
+         "mxclientsystem/mxui/mxui.js",
+         "widgets/pushNotifications/lib/PushNotification.js"
+      ]
+   },
+   "cachebust":"635689412670032000"
 }
 ```
-For more information on components file please read this documentation:
-[Mendix Components](https://world.mendix.com/display/refguide5/Customizing+Hybrid+Mobile+Apps)
 
-The components file should be included in the theme folder of your application.
+### Step 5 - Update index.html file
 
+Update `theme/index.html`to include the following reference to this javascript library:
 
-## Sending push notifications
+```
+<script type="text/javascript" src="widgets/pushNotifications/lib/PushNotification.js"></script>
+```
 
-In the module there are two operations to send messages. One which will queue the messages and the other to send immediately. If you are planning to send several push notifications to different devices at once, then I would suggest using the queued method. If however you simply want to test the sending of messages then just use the send immediate microflow.
+<img src="assets/images/implementation guide/Update index html file.PNG"/>
 
-In the module there is a devices page. This will list all of the devices that are currently registered to the application. If you want to send a message to one of these devices you can simply select a device and click send message.
+### Step 6 - Include Push Notification Snippet in the layouts
 
-To send a message without using the devices page, simply create a microflow that retrieves the device from a user account and create a message object with the attributes filled. Then simply pass this message to one of the microflows in the use me folder that sends either a message or a list of messages.  
+Include a push notification snippet on mobile and tablet layouts. You can do this by drag-and-dropping the `PushNotification_Snippet` snippet (located in `_USE ME/Snippets` folder in PushNotifications module) to your layout.
```

Comment:
'include in' , not on.
'located in *the* `_USE_ME.Snippets` folder in *the* PushNotifications module'